src/app/plugins/scores.plugin.ts

```diff
@@ -0,0 +1,125 @@
+import { RichEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import Constants from '../../common/constants';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+
+
+export class ScoresPlugin extends Plugin {
+  public name: string = 'NCAA Scores Plugin';
+  public description: string = 'Gets score of a football game.';
+  public usage: string = 'scores <team name>; ex scores UCF';
+  public permission: ChannelType = ChannelType.Public;
+  public pluginChannelName: string = Constants.Channels.Public.Sports;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const endpoint = 'http://www.espn.com/college-football/bottomline/scores';
+    let desiredTeamQuery = this._parseTeam(args || []);
+
+    //If no args, team is UCF by default
+    if (desiredTeamQuery === '') {
+      desiredTeamQuery = 'UCF';
+    }
+
+
+    try {
+      const response = await this.container.httpService.get(endpoint);
+      const games = response.data.split('?'); //Each game ends with a ?
+
```

Comment:
nit: saa

Reply:
Sorry what does SAA mean?

Reply:
same as above -- whatever the last comment was. and dont be sorry :)